src/main/java/bhs/devilbotz/utils/ShuffleboardManager.java

```diff
@@ -49,13 +48,13 @@ public ShuffleboardManager() {
               }
             });
 
-    gripperSetpoint =
-        driveTab
-            .add("Gripper Setpoint", Gripper.getAtSetpoint())
-            .withPosition(0, 1)
-            .withSize(1, 1)
-            .withWidget(BuiltInWidgets.kBooleanBox)
-            .getEntry();
+    // gripperSetpoint =
+    //     driveTab
+    //         .add("Gripper Setpoint", .getAtSetpoint())
+    //         .withPosition(0, 1)
+    //         .withSize(1, 1)
+    //         .withWidget(BuiltInWidgets.kBooleanBox)
+    //         .getEntry();
```

Comment:
Why are you disabling this?

Reply:
Because the getAtSetpoint method was static.  I did not want to have Shuffleboard try to display it if on practice bot.  I didn't have a good idea how to make it work after changing it to be class function instead of static.

Reply:
We might not even need the gripper setpoint on the shuffleboard, anyway.  We should probably discuss with @adipraj17 what information/controls would be most useful for him as a driver.